### DISCUSSION TITLE
ERROR - Failed to execute job

### DISCUSSION THREAD

**Discussion by YiiTing** [apache/airflow#40081] [Q&A]
Hi everyone,

I'm using Airflow 2.9.0 with the Localexecutor and I'm facing a strange issue.my task get some error as below.

> {standard_task_runner.py:110} ERROR - Failed to execute job 109 for task XXX ([Errno 2] No such file or directory: '/tmp/tmpioidmna3'; 1437842)

Any ideas about what to check for?

Thanks in advance!

**Comment by chaityacshah** [comment]
can you please share your code?

**Reply by YiiTing** [reply]
Hi,
i find error messages in scheduler log

> Failed to execute task module 'psutil' has no attribute 'Process'

so i reinstall airflow, and it can execution
thanks for your time
